### PR TITLE
[12.0][MIG] l10n_be_vat_reports: VAT Listing and VAT Intra

### PR DESCRIPTION
**Pull request by robinkeunen**
This migration is based on the v8 odoo standards reports for _VAT Listing_ and _VAT Intra_. The work was started by @andreparames at #57, improved at #74 for v10 and #75 for v9.

This time, I tried to include original v8 code as @sbidoul recommended (see 4e5eccc and 9ca1106).

Since reports have changed in v12, I refactored most of the code to avoid passing data to the report templates: everything is on the model.

### CODE REVIEW

**Comment by houssine78** [comment]
Hi @sbidoul do you think that this PR can be merged ?

**Review by remytms — APPROVED** [review]
LGTM

File: l10n_be_vat_reports/views/report_vatintraprint.xml
Comment (vvrossem) [inline l10n_be_vat_reports/views/report_vatintraprint.xml]: should be removed ?

File: l10n_be_vat_reports/views/report_vatintraprint.xml
Comment (vvrossem) [inline l10n_be_vat_reports/views/report_vatintraprint.xml]: should be removed?

File: l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.py
Comment (vvrossem) [inline l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.py]: should be solved or removed?

File: l10n_be_vat_reports/wizard/l10n_be_vat_intra_view.xml
Comment (vvrossem) [inline l10n_be_vat_reports/wizard/l10n_be_vat_intra_view.xml]: should be removed ?

**Review by vvrossem — COMMENTED** [review]
LGTM, but some commented lines need to be checked

**Comment by enricostano** [comment]
Hi @robinkeunen @houssine78 what's missing to get merged this? Can we help in any way?

**Comment by robinkeunen** [comment]
We were waiting for a review from @sbidoul but this module should probably be recoded in a cleaner way like in [this PR](https://github.com/OCA/l10n-belgium/pull/99)

**Comment by houssine78** [comment]
@robinkeunen this PR can be merged as it is as it a porting from previous version. At least we should take into account the comment from @vvrossem by removing commented lines. We use this PR in production for more than two years now. refactoring on this version would probably means never get it merged. Let's keep pragmatic.

**Comment by houssine78** [comment]
Hi @sbidoul could you have a quick look on this PR ? all the comments have been taken into account. 

thanks !

File: l10n_be_vat_reports/__manifest__.py
Comment (sbidoul) [inline l10n_be_vat_reports/__manifest__.py]: This comment is incorrect. BTW, you could simplify the copyright headers using the OCA template and also add Odoo and CoopIT Easy ?

**Comment by sbidoul** [comment]
I folks, sorry for the delay. I'd just be extra careful with the copyright attribution in all files, to avoid all possible trouble with Odoo.

The first commit is the same as Odoo 8.0, right ?

**Comment by robinkeunen** [comment]
Hi @sbidoul,

- I rebased for a cleaner history
- the first commit (6445398fbcb921c6f8c03f15c389b5b1774c0123) is the code from V8
- I reformated the comment section in `__manifest__` and other `.py` or `.xml` files but I left _tiny sprl_ as it is the company referred to in all v8 code
- On the other hand, I added Odoo SA under the author key of the manifest. I think that's enough :-)

**Comment by sbidoul** [comment]
Great thanks! Let's merge this at last.

/ocabot merge patch

**Pull request merged**
